test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) Registered app
+-- 4) Activated app
+--
+-- Steps: 
+-- 1) Send AlertManeuver mobile RPC from app with parametr imageType = "STATIC", HMI responds with 
+--    UNSUPPORTED_RESOURCE to Speak request
+--
+-- Expected:
+-- 1) App receives AlertManeuver response, with WARNINGS result code and success=true
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local params = {
+  ttsChunks = {
+    {
+      type = "TEXT",
+      text = "Text to Speak"
+    }
+  },
+  softButtons = {
+    {
+      systemAction = "DEFAULT_ACTION",
+      isHighlighted = true,
+      softButtonID = 5517,
+      type = "BOTH",
+      image = {
+        value = "icon.png",
+        imageType = "STATIC"
+      },
+      text = "Close"
+    },
+  }
+}
+
+--[[ Local Functions ]]
+local function sendAlertManeuver()
+  local mobileSession = common.mobile.getSession()
+  local hmi = common.hmi.getConnection()
+  local dataToHMI = {
+    appID = common.app.getHMIId()
+  }
+
+  local cid = mobileSession:SendRPC("AlertManeuver", params)
+  hmi:ExpectRequest("Navigation.AlertManeuver", dataToHMI)
+  :Do(function(_, data)
+    hmi:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  
+  hmi:ExpectRequest("TTS.Speak", { speakType = "ALERT_MANEUVER" })
+  :Do(function(_, data)
+    hmi:SendError(data.id, data.method, "UNSUPPORTED_RESOURCE", "Error message")
+    end)
+  mobileSession:ExpectResponse(cid, { success = true, resultCode = "WARNINGS" })
+end
+
+local function pTUpdateFunc(tbl)
+  tbl.policy_table.app_policies[common.app.getParams().fullAppID].groups = { "Base-4", "Navigation-1" }
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Update ptu", common.policyTableUpdate, { pTUpdateFunc })
```

Comment:
`PTU` is a `Policy Table Update` so we don't need `Update` word in step description, just `PTU` would be enough